src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -907,3 +1483,79 @@ def _select_items(self, items, item_type, sel_type):
             self._mapdl.vsel(sel_type, vmin=items, return_mapdl_output=True)
         else:
             raise ValueError(f'Unable to select "{item_type}"')
+
+
+class LegacyGeometry(Geometry):
+    """Legacy pythonic representation of MAPDL CAD geometry
+
+    Contains advanced methods to extend geometry building and
+    selection within MAPDL.
+    """
+
+    def keypoints(self) -> np.array:  # type: ignore
+        """Keypoint coordinates"""
+        return super().get_keypoints(return_as_array=True)
+
+    def lines(self) -> pv.PolyData:
+        """Active lines as a pyvista.PolyData"""
+        return super().get_lines()  # type: ignore
+
+    def areas(
+        self, quality=1, merge=False
+    ) -> Union[pv.PolyData, List[pv.UnstructuredGrid]]:
+        """List of areas from MAPDL represented as ``pyvista.PolyData``.
+
+        Parameters
+        ----------
+        quality : int, optional
+            quality of the mesh to display.  Varies between 1 (worst)
+            to 10 (best).
+
+        merge : bool, optional
+            Option to merge areas into a single mesh. Default
+            ``False`` to return a list of areas.  When ``True``,
+            output will be a single mesh.
+
+        Returns
+        -------
+        list of pyvista.UnstructuredGrid
+            List of ``pyvista.UnstructuredGrid`` meshes representing
+            the active surface areas selected by ``ASEL``.  If
+            ``merge=True``, areas are returned as a single merged
+            UnstructuredGrid.
```

Comment:
Should ``UnstructuredGrid`` be tagged as a literal, or can it be broken out to be text: unstructured grid?

Reply:
Roger.